### DISCUSSION TITLE
Where can I upload a video from my local disk?

### DISCUSSION THREAD

**Discussion by brian2354234** [pietrop/slate-transcript-editor#20] [Q&A] [answered]
https://pietropassarelli.com/slate-transcript-editor/?path=/story/slatetranscripteditor--demo

I didn't find any button to upload a video from my local disk? Can this program support voice and video recognization?

**Comment by pietrop** [comment, marked as answer]
HI @brian2354234 
This is a React component, to be used as part of a larger application. If you are looking for a ready to use solution to transcribe from your local disk you can checkout [autoEdit.io](www.autoEdit.io) (also free and open source) as it uses this component to display the transcript.
